CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,86 @@
+## Goal
+
+Our goal is to provide a space where it is safe for everyone to contribute to
```

Comment:
Does this refer explicitly to GitHub? I think we could say "space**s** where ... contribute to **and get support for** open-source software", if this sentence also includes the community.